compiler/src/iree/compiler/DispatchCreation/test/form_dispatch_regions.mlir

```diff
@@ -1331,3 +1331,53 @@ util.func @attention_rope_fusion(%arg0: tensor<10x20x30x50xbf16>,
 //  CHECK-SAME:         ins(%[[Q]], %[[K]], %[[V]]
 //       CHECK:     flow.return %[[ATTENTION]]
 //       CHECK:   util.return %[[DISPATCH]]
+
+// -----
+
+util.func public @avoid_illegal_consumer_fusion(%arg0: tensor<75600x5120xf32>) -> tensor<75600x1x5120xbf16> {
```

Comment:
It may be easier if you put `diamond` to the func name or the ASCII art to a comment. E.g., 

```
    [1]     [2]
    /  \     |
   /    \   [3]
 [E]   [4] <-'
   \   /
    [6]
```